Ch11/Ch11.md

```diff
@@ -728,23 +728,10 @@ from all the intermediate steps to get the final tally. Listing 11.14 shows the
 ```
 
 The `Cities/Details` index is interesting, in the sense that it shows off capabilities, but it isn't really that
-exciting. It also operates on a small dataset, even if it touches multiple collections. Let's modify the index
-by also having it operate on orders. 
-
-> **MapReduce indexes are also just indexes**
->
-> It bears repeating that a MapReduce index is also just an index. We focused heavily in this chapter on the MapReduce
-> portion of such indexes, but it's important to remember that they're also capable of doing everything that a 
-> map-only index can do. 
->
-> In other words, if we had a `Cities` collection that also had the coordinates of each city, we
-> would be able to modify the `Cities/Details` index to be a MultimapReduce index that also provides spatial
-> queries.
-> You can use facets on a MapReduce index, apply suggestions, run full text queries, etc. All too often, I see users
-> assume the MapReduce part of the index is where the functionality stops. In fact, this is where
-> it _begins_.
+exciting. It also operates on a small dataset, even if it touches multiple collections. 
+Let's modify the index by having it operate also on orders. 
 
-We'll first add another entry for each of the existing maps `OrderTotal = 0`. And we'll add the same logic for the reduce
+We'll first add another entry for each of the existing map: `OrderTotal = 0`. And we'll add the same logic for the reduce 
```

Comment:
` each of the existing map:`  <- Shouldn't this be maps, plural?

Reply:
yes, done.